libs/e2e/src/lib/describer/modelAssessment/causalAnalysis/describeCausalAnalysis.ts

```diff
@@ -1,33 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { visit } from "../../../../util/visit";
 import { Locators } from "../Constants";
 import { modelAssessmentDatasets } from "../datasets/modelAssessmentDatasets";
-import {
-  IModelAssessmentData,
-  RAINotebookNames
-} from "../IModelAssessmentData";
+import { IModelAssessmentData } from "../IModelAssessmentData";
 
 import { describeAggregateCausalAffects } from "./describeAggregateCausalAffects";
 
 const testName = "Causal Analysis";
 
 export function describeCausalAnalysis(
   datasetShape: IModelAssessmentData,
-  name?: keyof typeof modelAssessmentDatasets
+  name: keyof typeof modelAssessmentDatasets
```

Comment:
There is an assumption that if name is sent it is from OSS. If not, it is from AML, host will not be visited and runs tests directly.
Please make sure both cases are working as expected.

Reply:
same for all visits.